content/configuration/dev-server.md

```diff
@@ -129,6 +173,29 @@ To disable `contentBase`:
 contentBase: false
 ```
 
+Usage via the CLI
+
+```bash
+webpack-dev-server --content-base /path/to/content/dir
+```
+
+
+## `devServer.disableHostCheck`
```

Comment:
I think if it's an available option and it's noted by the CLI when running `webpack-dev-server -h` then it should be documented here as well. Your "THIS IS NOT RECOMMENDED..." is prominent enough that people should notice this, and if they choose to proceed anyway, so be it.

I'm going to do one last skim and then merge this. @SpaceK33z if you have a chance to review this later on and come across any issues -- please ping us so we can resolve in a separate PR.